community/modules/file-system/nfs-server/variables.tf

```diff
@@ -112,7 +122,7 @@ variable "local_mounts" {
     error_message = "Local mountpoints have to start with '/'."
   }
   validation {
-    condition     = length(var.local_mounts) > 0
-    error_message = "At least one local mount must be specified in var.local_mounts."
+    condition     = var.local_mounts != null
```

Comment:
nit. Can we set `nullable = false` instead ?

Reply:
`nullable = false` does 2 things:

1. The value can never be `null` _inside the module_ 
2. If a parent module of this module uses this as a child module and supplied `null` to `var.local_mounts`, then `var.local_mounts` takes on its default value

In the context of this PR, 1 is desirable and 2 is probably not desirable. That said, the way I read the [docs] suggests that if you remove the default value, set `nullable = false`, you might have the equivalent of the validation block.

https://developer.hashicorp.com/terraform/language/values/variables#disallowing-null-input-values

Having said _that_, if it works at all, it might not produce a useful error message. I believe that this error message should be improved to guide the user directly to `var.local_mount` or to web-based documentation about migrating to `var.local_mount`.